liquidprompt

```diff
@@ -1594,6 +1595,23 @@ _lp_as_text()
     echo -nE "$1" | sed -$_LP_SED_EXTENDED "s,$_LP_CLEAN_ESC,,g"
 }
 
+# Get length of a given string, after evaluating escape codes (\u for user, etc)
+_lp_length()
+{
+    typeset len evaluated_ps1
+    typeset printable_ps1=$(_lp_as_text "$1")
+
+    if $_LP_SHELL_bash; then
+        evaluated_ps1=$(printf '%s' "${printable_ps1@P}")
```

Comment:
This is a Bash-4.4+ feature:
https://github.com/bminor/bash/blob/76404c85d492c001f59f2644074333ffb7608532/CHANGES#L2400-L2401
Liquidprompt supports Bash-3.2+, so this is a no-go.

How to evaluate escape codes is a known problem in Liquidprompt. See:
* Terminal title: https://github.com/nojhan/liquidprompt/issues/609#issuecomment-751882825
* Ksh support: https://github.com/nojhan/liquidprompt/issues/608#issuecomment-645592979

On the surface it is simple, and should not be hard to fix, but I think there could be issues that spring up when we try to replace all escape codes with strings.

Anyway, that is on the top of my to-do list for v2.1. I don't see a way to make this PR work without that first, so you might need to wait...